`make devel-deps` is failed due to path of golint

`make devel-deps` failed as the followings.

```
$ make devel-deps
go get  github.com/golang/dep/cmd/dep
dep ensure
go get  github.com/golang/lint/golint \
	  github.com/haya14busa/goverage          \
	  github.com/mattn/goveralls              \
	  github.com/motemen/gobump               \
	  github.com/Songmu/goxz/cmd/goxz         \
	  github.com/Songmu/ghch                  \
	  github.com/tcnksm/ghr
package github.com/golang/lint/golint: code in directory /Users/autopp/go/src/github.com/golang/lint/golint expects import "golang.org/x/lint/golint"
make: *** [devel-deps] Error 1
```

It seems to be caused by the path of golint being changed.
https://github.com/golang/lint/issues/415

--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ deps:
 	dep ensure
 
 devel-deps: deps
-	go get ${u} github.com/golang/lint/golint \
+	go get ${u} golang.org/x/lint/golint \
 	  github.com/haya14busa/goverage          \
 	  github.com/mattn/goveralls              \
 	  github.com/motemen/gobump               \

